src/pybind/mgr/dashboard/controllers/cluster_configuration.py

```diff
@@ -3,14 +3,34 @@
 
 import cherrypy
 
-from . import ApiController, RESTController
+from . import ApiController, RESTController, ControllerDoc, EndpointDoc
 from .. import mgr
 from ..security import Scope
 from ..services.ceph_service import CephService
 from ..exceptions import DashboardException
 
 
+FILTER_SCHEMA = [{
+    "name": (str, 'Name of the cluster'),
+    "type": (str, 'Cluster type'),
+    "level": (str, 'level of info'),
+    "desc": (str, 'description of the configuration'),
+    "long_desc": (str, 'elaborated description'),
+    "default": (str, 'Default value set for the cluster'),
+    "daemon_default": (str, 'Default Daemon associated to the cluster'),
```

Comment:
```suggestion
    "daemon_default": (str, 'Daemon specific default value'),
```

https://docs.ceph.com/docs/master/dev/config/#default-values